.circleci/config.yml

```diff
@@ -37,7 +37,7 @@ commands:
                 fail_only: true
                 failure_message: ':fire: The $CIRCLE_JOB job has failed on the main branch! :fire:'
                 include_job_number_field: false
-                mentions: maxammann,st,UUZS7CMV0,UV99P8ETW
+                mentions: maxammann,st,UV99P8ETW
```

Comment:
If anyone wants to receive slack notifications for failing builds on main, let me know :) Right now, this just mentions @Taggotty, @maxammann and myself.